src/_guides/language/effective-dart/design_migrated.md

```diff
@@ -933,20 +938,31 @@ not intended to be invoked from Dart code and don't need a corresponding getter.
 [angular]: {{site.angulardart}}
 
 
-### AVOID returning `null` from members whose return type is `bool`, `double`, `int`, or `num`.
+### AVOID public `late final` fields.
+
+Unlike other final fields, a late final field *does* define a setter. If
```

Comment:
```suggestion
Unlike other final fields, a late final field without an initializing expression *does* define a setter. If
```

Reply:
😮 Today I learned.

Reworded and updated the guideline title too.